src/frameworks/drupal7/simplesaml.md

```diff
@@ -96,6 +101,22 @@ if (isset($_ENV['PLATFORM_PROJECT_ENTROPY'])) {
 }
 ```
 
+## Generate SSL certs (optional)
+
+Depending on your SimpleSAML Service Provider (SP), you may need to generate an SSL/TLS certificate to connect to the SP.  If so, you should generate the cert locally following the instructions in the [SimpleSAMLphp documentation](https://simplesamlphp.org/docs/stable/simplesamlphp-sp).  Whatever your resulting idP file is should be placed in the simplesamlphp/metadata directory.  The certificate should be placed in the `simplesamlphp/cert` directory.  (Create it if needed.)
```

Comment:
I think this should be "Depending on your Identity Provider (IdP)..." and "...connect to the IdP". The SP is the thing we are currently configuring.